xml/System.Globalization/CultureInfo.xml

```diff
@@ -73,106 +73,106 @@
+
+ Creating an instance of a <xref:System.Globalization.CompareInfo> class for a neutral culture is not recommended because the data it contains is arbitrary. To display and sort data, specify both the language and region. Additionally, the <xref:System.Globalization.CompareInfo.Name%2A> property of a <xref:System.Globalization.CompareInfo> object created for a neutral culture returns only the country and does not include the region.
+
+ The defined cultures have a hierarchy in which the parent of a specific culture is a neutral culture and the parent of a neutral culture is the invariant culture. The <xref:System.Globalization.CultureInfo.Parent%2A> property contains the neutral culture associated with a specific culture. Custom cultures should define the <xref:System.Globalization.CultureInfo.Parent%2A> property in conformance with this pattern.
+
+ If the resources for a specific culture are not available in the operating system, the resources for the associated neutral culture are used. If the resources for the neutral culture are not available, the resources embedded in the main assembly are used. For more information on the resource fallback process, see [Packaging and Deploying Resources](/dotnet/framework/resources/packaging-and-deploying-resources-in-desktop-apps).
+
+ The list of locales in the Windows API is slightly different from the list of cultures supported by .NET. If interoperability with Windows is required, for example, through the p/invoke mechanism, the application should use a specific culture that's defined for the operating system. Use of the specific culture ensures consistency with the equivalent Windows locale, which is identified with a locale identifier that is the same as <xref:System.Globalization.CultureInfo.LCID%2A>.
+
+ A <xref:System.Globalization.DateTimeFormatInfo> or a <xref:System.Globalization.NumberFormatInfo> can be created only for the invariant culture or for specific cultures, not for neutral cultures.
+
+ If <xref:System.Globalization.DateTimeFormatInfo.Calendar%2A?displayProperty=nameWithType> is the <xref:System.Globalization.TaiwanCalendar> but the <xref:System.Threading.Thread.CurrentCulture%2A?displayProperty=nameWithType> is not set to `zh-TW`, then <xref:System.Globalization.DateTimeFormatInfo.NativeCalendarName%2A?displayProperty=nameWithType>, <xref:System.Globalization.DateTimeFormatInfo.GetEraName%2A?displayProperty=nameWithType>, and <xref:System.Globalization.DateTimeFormatInfo.GetAbbreviatedEraName%2A?displayProperty=nameWithType> return an empty string ("").
+
+<a name="Custom"></a>
+## Custom cultures
+ In addition to the predefined cultures supported by the operating system and .NET, .NET supports three types of custom cultures:
+
+- New cultures that supplement the cultures available in the operating system or .NET. For example, an application could install a <xref:System.Globalization.CultureInfo> object that represents the `fj-FJ` (or Fijan (Fiji)) culture on a system.
```

Comment:
Please verify this change.

Reply:
can we remove the whole section and point at the OS doc insrtead [https://docs.microsoft.com/en-us/windows/win32/intl/custom-locales](https://docs.microsoft.com/en-us/windows/win32/intl/custom-locales). 

Please note this is supported on Windows OS only. We should be clear about that.